components/src/core/Drawer/DrawerNavGroup.tsx

```diff
@@ -189,81 +135,75 @@ function NavigationListItem(
     expanded = false,
     expandHandler = (): void => {}
 ): ReactNode {
-    const {
-        onClick,
-        title,
-        subtitle,
-        items: subItems,
-        divider: itemDivider,
-        itemID,
-        chevron: itemChevron,
-        collapseIcon,
-        expandIcon,
-    } = navItem;
-    // only allow icons for the top level items
-    const icon = !depth ? (navItem as NavItem).icon : undefined;
```

Comment:
Did we lose the check here for not allowing icons on nested items?

Reply:
Good catch -- looks like it might be some merge conflict misfortune thing